roles/hosts/tasks/type-block.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Get /etc/hosts file
+- name: Get /etc/hosts file  # noqa var-naming[no-reserved]
   ansible.builtin.slurp:
     src: "{{ hosts_file }}"
   register: hosts
```

Comment:
I'd prefer to actually rename this var, maybe `hosts_entries`? or `hosts_file_contents`?

Reply:
Sure can do this. If it is not necessary that the var is named "hosts"